src/electron/package_linux.action.sh

```diff
@@ -13,11 +13,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+BUILD_MODE=debug
+for i in "$@"; do
+    case $i in
+    --platform=*)
```

Comment:
Remove

Reply:
if i remove this it will error out if someone accidentally passes the --platform through. is that ok?

Reply:
Yes, that's fine. They shouldn't be passing the flag. It makes the reader think it's needed.